app/scripts/views/components/dataTable/tableView.js

```diff
@@ -164,26 +164,25 @@
       selectedGeneData.length = 0;
 
       if (geneData_) {
-        _.each(geneData_, function(item) {
+        _.each(geneData_, function(item, index) {
```

Comment:
I need index for each gene data, which is going to be unique id for the table row data.